ui/src/app/pages/workspace/workspace-edit-text.tsx

```diff
@@ -140,6 +140,12 @@ export const toolTipText = {
   cdrSelect: <div>The Curated Data Repository (CDR) is where research data from the <AouTitle/> is
     stored. The CDR is periodically updated as new data becomes available for
     use. You can select which version of the CDR you wish to query in this workspace.</div>,
+  tierSelect: <FlexColumn>
+    <div style={{fontWeight: 'bold'}}>Registered Tier dataset</div>
+    <div>This dataset includes surveys, electronic health records, biosamples, and physical measurements.</div>
+    <div style={{fontWeight: 'bold'}}>Controlled Tier dataset</div>
+    <div>This dataset contains expanded participant data, including genomics. Before you can access controlled tier data,
+    your institution will need to sign an amended agreement with the All of Us Data and Research Center.</div></FlexColumn>,
```

Comment:
Is this strictly true for Alpha?  (if not, do we care?)

Reply:
For Alpha, their institution will be "AouOps", and involves some out-of-band agreement, so I don't think it has bearing on them for Alpha (that said -it's fine to show this with the understanding that this is what prod users eventually should see).